control/monitor.go

```diff
@@ -98,7 +100,7 @@ func (m *monitor) Start(availablePlugins *availablePlugins) {
 	m.State = MonitorStarted
 }
 
-// stop the monitor
+// Stop the monitor
```

Comment:
Either 
Stop stops the monitor  (to keep with how you're doing everything else)
or 
Stops the monitor